tembo-cli/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 workspace = { members = ["temboclient", "tembodataclient"] }
 [package]
 name = "tembo-cli"
-version = "0.13.5"
+version = "0.13.6"
```

Comment:
should be a minor version since this is adding a new feature

Reply:
Good point @ChuckHend. Updated it.